p58.///Write a function to compute 5/0 and use try/except to catch the 
exceptions.///

code:

def compute():
    try:
        result = 5/0
    except ZeroDivisionError:
        print("Error: Division by zero!")
    else:
        print("Result:", result)